cetmix_tower_server/views/cx_tower_command_view.xml

```diff
@@ -120,7 +120,7 @@
                             <field
                                 name="code"
                                 attrs="{'required': [('action', '=', 'python_code')]}"
-                                widget="ace"
+                                widget="ace_tower"
```

Comment:
This works, need to add to the [path](https://github.com/DemchukM/cetmix-tower/blob/4bc940516cec0b56f26402f5e57cc122d7fc36b4/cetmix_tower_server/views/cx_tower_command_view.xml#L70) field too.